polyfill/test/timezone.mjs

```diff
@@ -197,6 +208,18 @@ describe('TimeZone', () => {
       }
     });
   });
+  describe('sub-minute offset', () => {
+    const zone = new Temporal.TimeZone('Europe/Amsterdam');
+    const inst = Temporal.Instant.from('1900-01-01T12:00Z');
+    const dtm = Temporal.DateTime.from('1900-01-01T12:00');
+    it(`${zone} has ID ${zone}`, () => equal(zone.id, `${zone}`));
```

Comment:
A feel a bit iffy about comparing two computed values, one of these should ideally be a constant, otherwise they could _both_ be the wrong yet the same and it would pass. That said, it's a very minor nit.

Reply:
This was already the case in the other tests surrounding this one, but maybe it's worth a follow up issue if you want to open one?